docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -64,31 +74,114 @@ Over the duration of a given microsimulation run, simulant attributes are dynami
+
+Residual confounding
+~~~~~~~~~~~~~~~~~~~~
+
+Getting realistic, individual-level characteristics from population-level data is difficult, and creates opportunity for residual confounding...
+
+Simulant agency
+~~~~~~~~~~~~~~~
+
+Unlike agent-based models, microsimulations do not necessarily program agency into individual simulants. This removes some of the real-world likeness of these models...
 
 .. todo::
 
-  Fill out this section with strengths and weakness of individual-based microsimulation. Could also add context about how microsim compares to agent-based simulation and the broader umbrella term of IBM.
+  Fill out this section with strengths and weakness of microsimulation. Could also add context about how microsim compares to agent-based simulation and the broader umbrella term of IBM.
+
+  Explanation of relationships between macroscopic population-level measures such as incidence rate and the corresponding microscopic hazard rates we use in our simulations, and how these relationships might affect our model design and V&V.
+
+  Cite [Allen-et-al-2019]_ and [Sorensen-et-al-2017]_ somewhere.
   
 
 How does Vivarium compare with other microsimulation tools?
 -----------------------------------------------------------
 
+Leveraging the Global Burden of Disease (GBD) Study
++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Based on GBD population-level estimates stratified by age, sex, time, and location, the IHME microsimulation framework creates a simulated population that mirrors the relevant qualities of a given real-world population of interest. Each individual simulant in the population is assigned health and demographic attributes that specify characteristics such as age, sex, disease, and risk factor status (for example, diabetes diagnosis, systolic blood pressure level, or body-mass index). In any given microsimulation analysis, there can be thousands, millions, or, theoretically, billions of simulants. Although the assigned health and demographic attributes are heterogeneous across individuals within the simulated population, when taken in aggregate they match the relevant real-world population-level data supplied by the GBD.
+
+The IHME platform is uniquely positioned to extract relevant input parameters from the vast repository of GBD population health data that are explicitly structured to quantify uncertainty by representing estimates as draws from Bayesian-derived joint posterior distributions.
+
+The IHME microsimulation platform, unlike most other microsimulation frameworks, additionally reduces variance between scenarios by appropriately controlling stochastic uncertainty through a common-random-numbers approach. In this procedure, for each simulant, the same randomly generated number is used in both baseline and intervention scenarios to dictate change or maintenance of health status across time steps.
```

Comment:
I don't think this belongs in this section. I think this is a pro of Vivarium vs other individual-based modeling tools.